TFNetworkLayer.py

```diff
@@ -1892,14 +1893,19 @@ def __init__(self, activation, with_bias=True, grad_filter=None,
     :param str forward_weights_init: see :func:`TFUtil.get_initializer`
     :param str recurrent_weights_init: see :func:`TFUtil.get_initializer`
     :param str|float bias_init: see :func:`TFUtil.get_initializer`
+    :param int|None out_feature_dim: if set, will convert feature dim of the input and output data
```

Comment:
It's not a dim, it's an axis. The parameter name should reflect that. E.g. like `out_feature_dim_axis`.

Reply:
Also the documentation ("feature dim axis", not "feature dim").